From: "Barak A. Pearlmutter" <barak+git@cs.nuim.ie>
Date: Thu, 6 Feb 2014 13:35:16 +0000
Subject: man macros

Avoid lintian error

qhull-bin: manpage-has-errors-from-man

to whit

  usr/share/man/man1/qhull.1.gz 750: warning: macro `QJn'' not defined
  usr/share/man/man1/qhull.1.gz 771: warning: macro `Qt'' not defined

and consequent lost text.

These result from a line starting with the single quote character, so
we Judiciously move line breaks to avoid that.
---
 html/qhull.man | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/html/qhull.man b/html/qhull.man
index 4ede521..038c9b6 100644
--- a/html/qhull.man
+++ b/html/qhull.man
@@ -746,8 +746,7 @@ not increase n beyond a certain value, and it stops after a certain number
 of attempts [see user.h].  Option 'QJ'
 selects a default value for n.  The output will be simplicial.  For
 Delaunay triangulations, 'QJn' sets 'Qbb' to scale the last coordinate
-(not if 'Qbk:n' or 'QBk:n' is set).
-'QJn' is deprecated for Voronoi diagrams.  See also 'Qt'.
+(not if 'Qbk:n' or 'QBk:n' is set).  'QJn' is deprecated for Voronoi diagrams.  See also 'Qt'.
 .TP
 Qm
 Only process points that would otherwise increase max_outside.  Other
@@ -768,8 +767,8 @@ Qs
 Search all points for the initial simplex.
 .TP
 Qt
-Triangulated output.  Triangulate all non\[hy]simplicial facets.  
-'Qt' is deprecated for Voronoi diagrams.  See also 'Qt'.
+Triangulated output.  Triangulate all non\[hy]simplicial
+facets.  'Qt' is deprecated for Voronoi diagrams.  See also 'Qt'.
 .TP
 Qv
 Test vertex neighbors for convexity after post\[hy]merging.
